packages/admin/src/Resources/Pages/ListRecords/Concerns/CanCreateRecords.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Filament\Resources\Pages\ListRecords\Concerns;
+
+use Filament\Pages\Actions\ButtonAction;
+use Filament\Pages\Actions\Modal;
+use Filament\Resources\Pages\Concerns\UsesResourceForm;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+
+trait CanCreateRecords
+{
+    use UsesResourceForm;
+
+    protected function hasCreateAction(): bool
+    {
+        if (! static::getResource()::canCreate()) {
```

Comment:
This can be a one liner - just return the condition without the `!`

Reply:
Hah - must have been a bad refactor. Thanks.